### ISSUE TITLE
Disable / Prevent Auto Focus❓ 

### ISSUE THREAD

**Issue by RWilliams-MicronAgritech** [cuvent/react-native-vision-camera#1140]
### Question

Hi,

Recently we've integrated with RNVC & replaced the deprecated RNC. Something that would be great for our use-case, would be to disable autofocus, is this possible out of the box in this library?

Ideally for our use-case, we would like to set a focus point (approx centre of the camera view), and have to remain in that area until otherwise.

I have checked the documentation, I can see you can manually set focus, which we are doing, however it still appears to be automatically trying to focus.

Any information on this would be gratefully  received.

Thanks!

### VisionCamera Version

2.13.2

### Additional information

- [ ] I am using Expo
- [X] I have read the [Troubleshooting Guide](https://mrousavy.github.io/react-native-vision-camera/docs/guides/troubleshooting)
- [X] I agree to follow this project's [Code of Conduct](https://github.com/mrousavy/react-native-vision-camera/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched for similar questions [in the issues page](https://github.com/mrousavy/react-native-vision-camera/issues?q=is%3Aopen+is%3Aissue+label%3A%22❓+question%22) as well as [in the discussions page](https://github.com/mrousavy/react-native-vision-camera/discussions) and found none.

**Comment by RWilliams-MicronAgritech** [comment]
Hold for now. I may have found it in the formats query 👍 .

**Comment by mrousavy** [comment]
Hey! Nice to hear, how's the migration going so far? Did you notice performance improvements, or other interesting features?

What exactly did you find in the Formats query? I think autofocus resets itself automatically after a given time (read that in the native Android/iOS docs a while ago), but I'm sure there must be a way to lock this. Then we'd need another method for unlocking again.

**Comment by RWilliams-MicronAgritech** [comment]
Hi Marc,
We've migrated for a few reasons, one of the main reasons was your Frame Processors, which would be great for us (we're yet to actually make use of this, as we are first migrating our basic app functionality). Migration has been very smooth, which is credit to the docs!

I noticed in the formats, theres a autoFocusSystem string, which I set too 'none'. It does however seem to have a slight side effect which is preventing the camera take images in landscape, and seems to force portrait (I need to investigate this a bit more to be sure).

Performance wise, I would say they're fairly similar in basic performance, however your module is much nicer to use in my opinion, documentation is well written. I guess the real test comes when I begin to use the frame processor.

Being able to "lock" & "unlock" the focus on the camera whilst it is active would be fantastic to be honest, I think that would be a positive improvement if it's possible. 

Thanks!
